### ISSUE TITLE
ATM7TTS v1.2.2 - bag of seeds - needs tweaking

### ISSUE THREAD

**Issue by dm8mydog** [AllTheMods/All-the-mods-7-Sky#422]
TLDR: bag of seeds - bag's loottable needs tweaking.

The bag of seeds is a drop that when right clicked will turn into some random seeds.

Some of the seeds it can turn into were otherwise disabled in the rest of the pack (they are uncraftable) and are unusable or broken even if they can be used.... (ie. the mystical ag. seeds essence to final item recipe is disabled)

This includes manasteel seeds, terrasteel seeds (botania), sulfur seeds (thermal), constantan seeds, bronze seeds (alloys), spirited crystal seeds (powah).... and quite a few more I'm sure. I have only opened a few bags of seeds and these were amongst the rewards.

cheers
